tools/cli/src/webpack/runtime-config.ts

```diff
@@ -122,6 +124,11 @@ export function getRuntimeConfig(buildFlags: BuildFlags): RuntimeConfig {
       : buildFlags.mode === 'development'
         ? true
         : currentBuildPreset.allowLocalWorkspace,
+    enableInfoModal: process.env.ENABLE_INFO_MODAL
```

Comment:
there is no need to add an env for it. it's not gonna be a optional feature after release